assets/src/settings-page/analytics.js

```diff
@@ -66,6 +67,9 @@ const vendorConfigs = {
 	},
 };
 
+/* Populating dropdown options by analytics vendors. */
+const AnalyticsVendorsOptions = Object.entries( AnalyticsVendors ).map( ( [ value, label ] ) => ( { value, label } ) );
```

Comment:
Why not store the entries in this way initially? Then there would be no need to do this conversion every time the page loads.

Reply:
Also, the array should be sorted alphabetically by the `label` in the data that is stored.

Reply:
Now, The entries are being stored in the required manner initially in the array.